deck/models.py

```diff
@@ -209,17 +210,23 @@ def user_already_voted(self, user):
         return self.votes.filter(user=user).exists()
 
     def user_can_vote(self, user):
-        can_vote = False
-        if self.user_already_voted(user) or \
-           (self.author == user and not self.event.author == user):
-            pass
-        elif self.event.allow_public_voting:
-            can_vote = True
-        elif user.is_superuser:
-            can_vote = True
-        elif self.event.jury.users.filter(pk=user.pk).exists():
-            can_vote = True
-        return can_vote
+        if self.user_already_voted(user):
```

Comment:
for me, i prefer the way that we have a local boolean var, like we had before this changes, but this works fines too and i am merging it

o/